src/Cli.Tests/ModuleInitializer.cs

```diff
@@ -19,6 +19,12 @@ public static void Init()
     {
         // Ignore the connection string from the output to avoid committing it.
         VerifierSettings.IgnoreMember<DataSource>(dataSource => dataSource.ConnectionString);
+        // Ignore the datasource files as that's unimportant from a test standpoint.
+        VerifierSettings.IgnoreMember<RuntimeConfig>(config => config.DataSourceFiles);
+        // Ignore the CosmosDataSourceUsed files as that's unimportant from a test standpoint.
```

Comment:
`files`? should it read: ` // Ignore the CosmosDataSourceUsed property as that's unimportant from a test standpoint.` ?

Reply:
fixed this. my bad.